releases/release-7.1.3.md

```diff
@@ -0,0 +1,211 @@
+---
+title: TiDB 7.1.3 Release Notes
+summary: 了解 TiDB 7.1.3 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 7.1.3 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：7.1.3
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v7.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v7.1.3#version-list)
+
+## 兼容性变更
+
+<!-- **tw:@qiancai** 1 -->
+
+- (dup): release-6.5.6.md > 兼容性变更 - 在安全增强模式 (SEM) 下禁止设置 [`require_secure_transport`](https://docs.pingcap.com/zh/tidb/v6.5/system-variables#require_secure_transport-从-v610-版本开始引入) 为 `ON`，避免用户无法连接的问题 [#47665](https://github.com/pingcap/tidb/issues/47665) @[tiancaiamao](https://github.com/tiancaiamao)
+- (dup): release-6.5.6.md > 兼容性变更 - 经进一步的测试后，TiCDC Changefeed 配置项 [`case-sensitive`](/ticdc/ticdc-changefeed-config.md) 默认值由 `true` 改为 `false`，即默认情况下 TiCDC 配置文件中涉及的表名、库名大小写不敏感 [#10047](https://github.com/pingcap/tiflow/issues/10047) @[sdojjy](https://github.com/sdojjy)
+- (dup): release-6.5.6.md > 兼容性变更> TiCDC Changefeed 新增以下配置项： - [`compression`](/ticdc/ticdc-changefeed-config.md)：你可以设置 redo log 文件的压缩行为 [#10176](https://github.com/pingcap/tiflow/issues/10176) @[sdojjy](https://github.com/sdojjy)
+- (dup): release-6.5.6.md > 兼容性变更> TiCDC Changefeed 新增以下配置项： - [`encoding-worker-num`](/ticdc/ticdc-changefeed-config.md) 和 [`flush-worker-num`](/ticdc/ticdc-changefeed-config.md)：你可以根据不同的机器规格，设置 redo 模块不同的并发参数 [#10048](https://github.com/pingcap/tiflow/issues/10048) @[CharlesCheung96](https://github.com/CharlesCheung96)
+- (dup): release-6.5.6.md > 兼容性变更> TiCDC Changefeed 新增以下配置项： - [`sink.cloud-storage-config`](/ticdc/ticdc-changefeed-config.md)：你可以设置同步数据到对象存储时自动清理历史数据的功能 [#10109](https://github.com/pingcap/tiflow/issues/10109) @[CharlesCheung96](https://github.com/CharlesCheung96)
+- 将自动支持平滑升级功能修改成通过 `/upgrade/start` 和 `upgrade/finish` HTTP 接口支持此功能  @[zimulala](https://github.com/zimulala)
```

Comment:
Removed this note because the change has already been made in v7.1.2:

https://docs.pingcap.com/zh/tidb/dev/release-7.1.2
![image](https://github.com/pingcap/docs-cn/assets/79440533/62eed712-09dd-40a8-a4b1-9f2f7412fcaf)